test/unit/utils/test_attrdict.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from robot.utils.asserts import assert_equal, assert_not_equal, assert_raises
+from Selenium2Library.utils import AttrDict
+try:
+    from robot.utils import IRONPYTHON
+except ImportError:  # New in RF 2.9
+    import sys
+    IRONPYTHON = sys.platform == 'cli'
+    pass
+
+
+class TestAttrDict(unittest.TestCase):
+
+    def setUp(self):
+        self.dd = AttrDict([('z', 1), (2, 'y'), ('x', 3)])
```

Comment:
It's a good idea to have tests for `AttrDict` and copying Robot's tests for `DotDict` is fine too. `dd` is a bit weird name for `AttrDict` instance, though.